cmd/mcp-broker-router/main.go

```diff
@@ -127,9 +132,17 @@ func main() {
 		return
 	}
 	ctx := context.Background()
-	brokerServer, mcpBroker, mcpServer := setUpBroker(mcpBrokerAddrFlag, enforceToolFilteringFlag)
+	var jwtSessionMgr *session.JWTManager
+	if jwtSigningKeyFlag != "" {
```

Comment:
The signing key is optional?
What's the expected behaviour if no key is specified, and expected value for the session id header?

Reply:
It has a default value that shouldn't really be used other than locally. But I think we can fail if not set at all and log if set to the default 👍